Foundation/NSURLSession/http/EasyHandle.swift

```diff
@@ -168,6 +171,20 @@ extension _EasyHandle {
         let protocols = (CFURLSessionProtocolHTTP | CFURLSessionProtocolHTTPS)
         try! CFURLSession_easy_setopt_long(rawHandle, CFURLSessionOptionPROTOCOLS, protocols).asError()
         try! CFURLSession_easy_setopt_long(rawHandle, CFURLSessionOptionREDIR_PROTOCOLS, protocols).asError()
+        #if os(Android)
+            // See https://curl.haxx.se/docs/sslcerts.html
+            // For SSL to work you need "cacert.pem" to be accessable
+            // at the path pointed to by the URLSessionCAInfo env var.
```

Comment:
Since you ask... :P

* These should probably be named `_CAInfoFilePath` and `setCAInfoFilePath`, as it's not the file but the file path (versus, say, the file URL).
* Style nit: unsure why you're indenting this block between `#if` and `#endif` but not any of the others you added.
* Style nit: why have a blank line [here](https://github.com/apple/swift-corelibs-foundation/pull/1103/files#diff-4bb7c5c7e89bd3e0c72cceca47b502c8R637) and before the closing brace?
* Nit: if you're going to log instead of print from `printLibcurlDebug`, shouldn't it be `logLibcurlDebug`?